src/catalog/src/memory/objects.rs

```diff
@@ -448,7 +446,7 @@ pub struct Source {
     pub desc: RelationDesc,
     pub timeline: Timeline,
     pub resolved_ids: ResolvedIds,
-    pub custom_logical_compaction_window: Option<Duration>,
+    pub custom_logical_compaction_window: Option<CustomCompactionWindow>,
```

Comment:
For these objects, what's the difference between `None` and `Some(CustomCompactionWindow::Default)`?

Reply:
none means the user did not specify a custom window in the syntax. maybe customcompactionwindow::default should not exist then since it's implied by none!